docs/contributing.rst

```diff
@@ -0,0 +1,10 @@
+Contributing to SPRAS
+=====================
+
+We welcome contributions to SPRAS! If you would like to contribute, please follow the guidelines below.
+
+Adding New PRMs
```

Comment:
I'd like to avoid the PRM algorithm in the docs. We could call these "Algorithms" generically if space is limited or "Network Algorithims" if we have a little more space.

@annaritz what do you think?

Reply:
I'll update the template for now. Once we merge this scaffolding, anyone is welcome to adapt the placeholder text I'm using throughout.